### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by LazarusAA**
Closes Issue https://github.com/ZencypherSolutions/semaphore-stellar-contracts/issues/5

This pull request adds a CONTRIBUTING.md file to the repository, outlining guidelines for contributors to follow when creating issues, forking the repo, and naming branches and issues.

Feel free to let me know if there’s anything you'd like me to adjust!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+Thank you for considering contributing to this project! We appreciate your time and effort in helping improve our work. Below are guidelines to streamline the contribution process, so please read them carefully. And don’t forget to join our [Telegram](https://t.me/+-9623JNgGjEyNzI5) for updates and community support!
+
+Following these guidelines shows respect for the maintainers’ time, and they’ll reciprocate by giving thoughtful attention to your issues, reviewing your changes, and supporting you in finalizing pull requests.
+
+### Check existing issues
+
+Before you start contributing, please check the [Issue Tracker](https://github.com/ZencypherSolutions/semaphore-stellar-contracts/issues) to see if there are any existing issues that match what you are intending to do. If the issue doesn't exist, please create it.
+
+If you are creating a new issue, please provide a descriptive title and detailed description. If possible, include a code sample or an executable test case demonstrating the expected behavior that is not occurring.
```
Comment (bitfalt) [inline CONTRIBUTING.md]: Add a new paragraph being more explicit that they can create issues for new features which would then be analyzed by the maintainers.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+### Fork and Clone the Repository
+
+Once you've found an issue to work on, the next step is to fork the Semaphore Stellar repo and clone it to your local machine. This is necessary because you probably won't have push access to the main repo.
```
Comment (bitfalt) [inline CONTRIBUTING.md]: Remove the probably since they won't have access.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+When you're ready to start coding, create a new branch on your cloned repo. It's important to use a separate branch for each issue you're working on. This keeps your changes separate in case you want to submit more than one contribution.
+
+Please use meaningful names for your branches that clearly indicate the purpose of your work. Since this is a documentation repository, use prefixes such as `docs` or `research`. For example, if you're working on a **CONTRIBUTING.md** file, you might name your branch `docs/contributing-guide`.
+
+As you're making changes, make sure you follow the coding conventions used throughout the Semaphore Stellar project. Consistent code style makes it easier for others to read and understand your code.
+
+## Submitting a Pull Request
+
+Once your changes are ready, commit them and push the branch to your forked repo on GitHub. Then you can open a pull request from your branch to the `main` branch of the Semaphore Stellar repo.
```
Comment (bitfalt) [inline CONTRIBUTING.md]: Be more explicit indicating the docs repository to avoid any confusion.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+When you submit the pull request, please provide a clear, detailed description of the changes you've made. If you're addressing a specific issue, make sure you reference it in the description.
+
+Your pull request will be reviewed by the maintainers of Semaphore Stellar. They may ask for changes or clarification on certain points. Please address their comments and commit any required changes to the same branch on your repo.
+
+## Final Notes
+
+Again, thank you for considering to contribute to Semaphore Stellar. Your contribution is invaluable to us. We hope this guide makes the contribution process clear and answers any questions you might have. If not, feel free to ask on [Telegram](https://t.me/+-9623JNgGjEyNzI5).
```
Comment (bitfalt) [inline CONTRIBUTING.md]: Remove again, just leave everything after thank you.

**Review by bitfalt — CHANGES_REQUESTED** [review]
Looks good overall, just a few changes!

**Comment by LazarusAA** [comment]
Completed the changes, please review.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+Following these guidelines shows respect for the maintainers’ time, and they’ll reciprocate by giving thoughtful attention to your issues, reviewing your changes, and supporting you in finalizing pull requests.
+
+### Check existing issues
+
+Before you start contributing, please check the [Issue Tracker](https://github.com/ZencypherSolutions/semaphore-stellar-contracts/issues) to see if there are any existing issues that match what you are intending to do. If the issue doesn't exist, please create it.
+
+### Create new issues
+
+If you are creating a new issue for fixes or features, please provide a descriptive title and detailed description. If possible, include a code sample or an executable test case demonstrating the expected behavior that is not occurring. This issues will then be analyzed by the maintainers and give you the green light to continue.
```
Comment (evgongora) [inline CONTRIBUTING.md]: Please just leave the "If you are creating a new issue", change "this issue" for "the issue"

**Review by evgongora — COMMENTED** [review]
LGTM! Good work :)

**Pull request merged**